docs/user-guide/data/work-on-data.md

```diff
@@ -0,0 +1,242 @@
+
+![Select 3 resources and compare them.](../../assets/images/compare-resource-select.png)*Example: Select 3 resources and compare them.*
+
+To keep in mind: The more you compare, the smaller the resource viewer for each will be. You can deselect one or more resources to remove them from the compare viewer.
+
+![Visualise your selected resources side-by-side, e.g. 3 resources.](../../assets/images/compare-resource-view.png)*Visualise your selected resources side-by-side. You can compare 2 images with an object without representation for instance.*
+
+---
+
+## Add new data
+
+&#9888; You must have the granted permission to proceed.
+
+### Add a resource
+
+See section [Create new resource instances](https://docs.dasch.swiss/latest/DSP-APP/user-guide/data/#create-new-resource-instances).
+
+### Add a property value
+
+You can create a new property value **if your data model allows it**. The value should have a cardinality of 0-1, 0-N or, 1-N.
+A "plus" button is displayed to the right of the label or after the first value field. You fill in the form field and save the changes by clicking on the floppy disk button. You can undo (back arrow button) or cancel the changes (x button).
+
+![Add a new property value.](../../assets/images/add-value.png)*Add a new property value.*
+
+![Save or undo the changes.](../../assets/images/add-new-text-value.png)*Save or undo the changes you made.*
+
+If your property is linked to another resource, you can search for an existing value by typing the first letter of the resource label you are looking for. You can also create a new resource instance by clicking on the first option in the drop-down menu, `Create new: xxx`.
+
+![Add a new linked resource value.](../../assets/images/add-new-resource-value.png)
+
+### Add an image region
+
+To create a new region on an image, you click on the button Draw a region in the toolbar, then select the region you want on the image.
+
+![Select a region.](../../assets/images/region_draw.png)*Select a region on the image with the drawing tool.*
+
+Enter the information about the region, an informative label and a description as a comment. You can change the color.
+
+![Fill in the form.](../../assets/images/region_fillin_form.png)*Fill in the form about the selected region.*
+
+The new region information is displayed after the image viewer. You can edit or delete if you have the granted permissions. A region has also an AKR URL you can copy and share.
+
+![Region created.](../../assets/images/region_new_annotation.png)*A new region has been created.*
+
+---
+
+## Edit your data
+
+&#9888; You must have the granted permission to proceed.
+
+### Edit a resource
+
+To edit the resource label, you reach the edit form by opening the resource menu on the right-side of the toolbar and clicking on `Edit label`.
```

Comment:
To edit the resource label, ~~you reach the edit form by opening~~ **open** the resource menu on the right-side of the toolbar and ~~clicking~~ **click** on `Edit label`.